mono/mini/exceptions-amd64.c

```diff
@@ -273,7 +274,12 @@ mono_arch_get_call_filter (MonoTrampInfo **info, gboolean aot)
 		if (AMD64_IS_CALLEE_SAVED_REG (i))
 			amd64_pop_reg (code, i);
 
+#if TARGET_WIN32
+	amd64_lea_membase (code, AMD64_RSP, AMD64_RBP, 0);
+	amd64_pop_reg (code, AMD64_RBP);
+#else
```

Comment:
The windows x64 ABI only handle a fixed set of epilog patterns that must be followed, so even if the result will be the same as using amd64_leave, it is not an instruction sequence that is approved to be part of an x64 Windows compliant epilog. Short story:

No FP:
add RSP, fixed-allocation-size  
pop's matching prolog pushes
ret  

FP:
lea      RSP, fixed-allocation-size – 128[RBP]  
pop's matching prolog pushes
ret

These are the only legal forms for an epilog. It must consist of either an add RSP,constant or lea RSP,constant[FPReg], followed by a series of zero or more 8-byte register pops and a return or a jmp

More details:

https://msdn.microsoft.com/en-us/library/tawsa7cb.aspx